client-app/src/Components/Login.js

```diff
@@ -54,67 +38,71 @@ const Login = props => {
     };
 
     return (
-        <div>
-            <form onSubmit={handleSubmit} className="my-5 container">
-                <div className="mb-3">
+        <div className="login-container">
+            <form onSubmit={handleSubmit} className="login-form">
+                <div className="form-group">
```

Comment:
@miabu-pashh It is fine you improvised the Login form, but we have another issue specifically for improvising login form. Please stick to resolving one issue in a single PR